packages/cli-plugin-metro/src/tools/loadMetroConfig.ts

```diff
@@ -89,17 +97,32 @@ export default async function loadMetroConfig(
   // @ts-ignore resolveConfig return value is mistyped
   logger.debug(`Reading Metro config from ${projectConfig.filepath}`);
 
-  try {
-    require.resolve('@react-native/metro-config', {
-      paths: [ctx.root],
-    });
-  } catch (e) {
+  if (
+    !/['"']@react-native\/metro-config['"']/.test(
+      fs
+        // @ts-ignore resolveConfig return value is mistyped
+        .readFileSync(projectConfig.filepath, 'utf8'),
+    )
+  ) {
     logger.warn(
-      "From React Native 0.72, your 'metro.config.js' file should " +
-        "extend '@react-native/metro-config', however it's not present in your " +
-        "project's devDependencies. Please install '@react-native/metro-config'.",
+      'From React Native 0.72, your metro.config.js file should extend' +
+        "'@react-native/metro-config'. Please see the React Native 0.72 " +
+        'changelog, or copy the template at:\n' +
+        'https://github.com/facebook/react-native/blob/main/packages/react-native/template/metro.config.js',
```

Comment:
Thanks for adding a reference to copy the config

Reply:
Ideal link is once we have a URL for the changelog/upgrade helper :)